SQ crashes on sample data on my laptop

In trying to reproduce a user's issue, I found that Scenic Quality crashes for me on my laptop when running it against the sample data:

```python
07/24/2023 14:25:12  (natcap.invest.scenic_quality.scenic_quality) scenic_quality._determine_valid_viewpoints(512) INFO Layer structures_clipped has 35 features
07/24/2023 14:25:12  (taskgraph.Task) Task.add_task(706) ERROR Something went wrong when adding task determine_valid_viewpoints (4), terminating taskgraph.
Traceback (most recent call last):
  File "/Users/jdouglass/workspace/natcap/invest/env310/lib/python3.10/site-packages/taskgraph/Task.py", line 674, in add_task
    new_task._call()
  File "/Users/jdouglass/workspace/natcap/invest/env310/lib/python3.10/site-packages/taskgraph/Task.py", line 1093, in _call
    payload = self._func(*self._args, **self._kwargs)
  File "/Users/jdouglass/workspace/natcap/invest/src/natcap/invest/scenic_quality/scenic_quality.py", line 553, in _determine_valid_viewpoints
    raise AssertionError(
AssertionError: Feature 0 is not a Point geometry. Features must be a Point.
Traceback (most recent call last):
  File "/Users/jdouglass/workspace/natcap/invest/sq.py", line 28, in <module>
    natcap.invest.scenic_quality.scenic_quality.execute(model_args)
  File "/Users/jdouglass/workspace/natcap/invest/src/natcap/invest/scenic_quality/scenic_quality.py", line 346, in execute
    valid_viewpoints_task = graph.add_task(
  File "/Users/jdouglass/workspace/natcap/invest/env310/lib/python3.10/site-packages/taskgraph/Task.py", line 674, in add_task
    new_task._call()
  File "/Users/jdouglass/workspace/natcap/invest/env310/lib/python3.10/site-packages/taskgraph/Task.py", line 1093, in _call
    payload = self._func(*self._args, **self._kwargs)
  File "/Users/jdouglass/workspace/natcap/invest/src/natcap/invest/scenic_quality/scenic_quality.py", line 553, in _determine_valid_viewpoints
    raise AssertionError(
AssertionError: Feature 0 is not a Point geometry. Features must be a Point.
```

--- a/src/natcap/invest/scenic_quality/scenic_quality.py
+++ b/src/natcap/invest/scenic_quality/scenic_quality.py
@@ -549,10 +549,10 @@ def _determine_valid_viewpoints(dem_path, structures_path):
 
             # Coordinates in map units to pass to viewshed algorithm
             geometry = point.GetGeometryRef()
-            if geometry.GetGeometryType() != ogr.wkbPoint:
+            if geometry.GetGeometryName() != 'POINT':
                 raise AssertionError(
-                    f"Feature {point.GetFID()} is not a Point geometry. "
-                    "Features must be a Point.")
+                    f"Feature {point.GetFID()} must be a POINT geometry, "
+                    f"not {geometry.GetGeometryName()}")
 
             viewpoint = (geometry.GetX(), geometry.GetY())
 

--- a/tests/test_scenic_quality.py
+++ b/tests/test_scenic_quality.py
@@ -126,7 +126,8 @@ class ScenicQualityTests(unittest.TestCase):
         with self.assertRaises(AssertionError) as cm:
             scenic_quality._determine_valid_viewpoints(
                 dem_path, viewpoints_path)
-        self.assertIn('Feature 1 is not a Point geometry', str(cm.exception))
+        self.assertIn('Feature 1 must be a POINT geometry, not LINESTRING',
+                      str(cm.exception))
 
     def test_exception_when_no_structures_aoi_overlap(self):
         """SQ: model raises exception when AOI does not overlap structures."""

